jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/PATCHSpec.scala

```diff
@@ -0,0 +1,77 @@
+package com.cerner.beadledom.jaxrs
+
+import com.cerner.beadledom.jaxrs.provider.{FakeModel, FakeResource}
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider
+
+import org.jboss.resteasy.core.Dispatcher
+import org.jboss.resteasy.mock.{MockDispatcherFactory, MockHttpRequest, MockHttpResponse}
+import org.jboss.resteasy.plugins.server.resourcefactory.POJOResourceFactory
+import org.scalatest._
+import org.scalatest.mock.MockitoSugar
+
+import java.io._
+
+import scala.collection.JavaConverters._
+import javax.ws.rs.core.MediaType
+
+/**
+  * Spec tests for {@link PATCH}.
+  *
+  * @author Eric Christensen
+  */
+class PATCHSpec extends FunSpec with BeforeAndAfterAll with ShouldMatchers with MockitoSugar {
```

Comment:
I think it should be a separate test directly in the `describe("Proxied Clients")` block that tests only that patch works.